docs/architecture/adr-0014-api-documentaton-revisited.md

```diff
@@ -0,0 +1,104 @@
+# API documentation ADR
+## Context
+
+The goal isn't to make this super complicated, but give us a basis for building out more 
+full-fledged documentation as the project goes -- beyond just listing API endpoints 
+(see Auth0's docs section as inspiration https://auth0.com/docs).
+
+### Frameworks to choose from
+#### Spectacle
+##### Pros
+* Very flexible styling
+* Actively maintained, and looks good
+* Delivered assets are a static site
+* Low maintenance
+##### Cons
+* Only supports generating API documentation - our requirements are along the lines of a CMS-lite type system, where code and api documentation can live side by side on the same site
+
+#### Angular 1.X
+##### Pros
+* We know it already, so it would be quick to get up and running
+* Probably "good enough" for all intents and purposes. This is the quickest and easiest solution
+##### Cons
+* We are planning on eventually moving away from it
+* Angular 1.X libraries are kind of in limbo right now - We've found that quite 
+a few frameworks are no longer being actively maintained, so we have to bring the 
+code into our own repository for maintenance purposes.  New angular libraries will
+also likely only be written for angular 2+, possibly limiting our options down the road to
+projects which only update with bugfixes.
+* Not auto-generated
+
+#### Angular 2
+##### Pros
+* Lets us experiment with Angular 2 before we move decide to move one of our larger projects over
+* Aligns with team objectives
+* Performance is aligned with the other major frameworks
+* Angular CLI seems pretty solid.  I haven't run into any of the usual webpack issues using it.
+  There is now an RC.0, which means that no project code changes should be required in order to 
+  upgrade to v1
+* Typescript is amazing
+* Supports Redux
```

Comment:
This and the Typescript point don't seem to distinguish the frameworks; all of them support Redux except maybe Angular 1.x, and all of them seem to support Typescript, too.